core/src/main/java/uk/ac/bbsrc/tgac/miso/core/security/SecurableByProfile.java

```diff
@@ -55,6 +55,12 @@
    * @throws SecurityException
    *           when the parent SecurityProfile could not be inherited
    */
-  void inheritPermissions(SecurableByProfile parent) throws SecurityException;
+  public default void inheritPermissions(SecurableByProfile parent) throws SecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.SecurityException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)